ports/stm/supervisor/internal_flash.h

```diff
@@ -35,6 +35,8 @@
 #ifdef STM32F401xE
 #define STM32_FLASH_SIZE 0x80000 //512KiB
 #define INTERNAL_FLASH_FILESYSTEM_SIZE 0xC000 //48KiB
+#define INTERNAL_FLASH_FILESYSTEM_START_ADDR 0x08004000
+#define INTERNAL_FLASH_FILESYSTEM_NUM_BLOCKS (INTERNAL_FLASH_FILESYSTEM_SIZE / FILESYSTEM_BLOCK_SIZE)
```

Comment:
`INTERNAL_FLASH_FILESYSTEM_NUM_BLOCKS` appears to be the same everywhere? Could you factor this out?